tests/apps/lit/test_lit_api.py

```diff
@@ -5,6 +5,8 @@
 from django.urls import reverse
 from rest_framework.test import APIClient
```

Comment:
I think we also need to add tests to the hawc_client too for these endpoints; I forgot to do so on the bioassay create API, will need to add...